bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala

```diff
@@ -288,7 +288,9 @@ object ITCRequest {
     }
 
     // Get Normalization info
-    val normBand = r.enumParameter(classOf[WavebandDefinition])
+    val bandName = r.parameter("WavebandDefinition")
+    val normBand = MagnitudeBand.all.find(_.name == bandName)
+    if (normBand.isEmpty) throw new IllegalArgumentException(s"Unsupported wave band $bandName")
```

Comment:
While we're throwing exceptions, I suppose you could just shed the `Option` immediately with

````scala
val normBand = MagnitudeBand.all.find(_.name == bandName).getOrElse(sys.error(s"Unsupported wave band $bandName")
````
Then you avoid the `normBand.get` later on.  Or maybe it needs to be `IllegalArgumentException`?